### PR TITLE
Add contributing, license thing, updates to readme

### PR DESCRIPTION
**Pull request by trumanshuck**
@go-between/folks 

Alright, adds a [source available](https://en.wikipedia.org/wiki/Source-available_software) license from [https://commonsclause.com/](https://commonsclause.com/).  Adds a CONTRIBUTING.md doc.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+3.  Merge away or wait for approval (depending on your access to the github repository itself)
+
+# Contributors
+
+- Dan Lavin
+- Jason Sisk
+- Sean Dennison
+- Truman Shuck
+
+# Source Available
```
Comment (trumanshuck) [inline CONTRIBUTING.md]: Does this section feel weird or wrong?  I can't decide if it's important to say or if it makes us greedy takey-takers.  I think that if this thing actually starts making money that we should cut off 10% revenue to contribute to the open source projects that we use somehow.

File: LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Without limiting other conditions in the License, the grant of rights under the License will not include, and the License does not grant to you, the right to Sell the Software.
+
+For purposes of the foregoing, “Sell” means practicing any or all of the rights granted to you under the License to provide to third parties, for a fee or other consideration (including without limitation fees for hosting or consulting/ support services related to the Software), a product or service whose value derives, entirely or substantially, from the functionality of the Software. Any license notice or attribution required by the License must also include this Commons Clause License Condition notice.
+
+Software: Musicbox.fm
+
+License: Apache 2.0
+
+Licensor: Truman Shuck
```
Comment (trumanshuck) [inline LICENSE.txt]: We'll change this to Go Between® LLC or whatever we end up incorporating.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+The Musicbox is run as a source-available software product in order to promote the open sharing of knowledge and the ability to find companionship through music while allowing its development team to feel secure in their existence within the our shared framework of capitalism and so provide for their dependent cats, children, and personal needs.
```
Comment (DLavin23) [inline CONTRIBUTING.md]: Typo: `...secure in their existence within the _our_ shared framework...`

File: CONTRIBUTING.md (hunk 1 above)
Comment (DLavin23) [reply CONTRIBUTING.md]: If it reads weird, it's probably the legalese/language—not that the sentiment is wrong or greedy. Our primary objective is still to build a profitable product. Arguably, making the source available and free to use locally **is** an example of one such open source contribution. I'm thinking this section will just have to evolve if we ever get to the point where there are some really serious and frequent contributors.

**Pull request merged**